lib/valkyrie/persistence/fedora/persister/orm_converter.rb

```diff
@@ -556,11 +556,7 @@ def blacklist?(key)
           # @param [Object] values
           # @return [Array<Object>]
           def cast_array(values)
-            if values.is_a?(Time)
-              [values]
-            else
-              Array(values)
-            end
+            Array(values)
```

Comment:
This used to be required because if you do Array(values) you got a weird array back, not just a Time wrapped in an array. No longer an issue with DateTime?

Reply:
It seems perfectly happy to return `[DateTime]` rather than all its components exploded.